### ISSUE TITLE
Enable user or admin to set .mm as default format

### ISSUE THREAD

**Issue by nursoda** [ACTom/files_mindmap#166]
I noticed that the app handles the .mm format nicely. Graphs created using the Freeplane desktop program may be viewed and edited. However, Freeplane can neither open nor import KityMinder (.km) files. Since I prefer to use locally installed programs but sometimes need to create a chart collaboratively, my workaround is to create an empty mindmap in Freeplane and edit that file in the Nextcloud web interface.

My request is to be able to create a new Mindmap in .mm format right from the "Create a new Document" (+) menu.

**Comment by Uatschitchun** [comment]
Tried this. Am able to edit the .mm Map, but edits won't get saved!?

**Comment by nursoda** [comment]
> Tried this. Am able to edit the .mm Map, but edits won't get saved!?

I confirm this on NC24 with Mindmap 0.0.26: An existing .mm-mindmap is displayed correctly and can be edited but is not saved. So .mm is import/display only? If so, the user should be noted! Otherwise all work done on the mindmap is lost. On the other hand: As it is displayed – shouldn't it be able to save it as .km at least (and offer to do just that)?

**Comment by vbspam** [comment]
I confirm the same. I would like to support this feature request :), this would make the tool great.

**Comment by AK-CCM** [comment]
I'm new to the Nextcloud universe. I've tried your great app because I'm using mindmaps often to sort and visualize my thoughts. At first I was very happy a .mm mindmap could be opened and edited if I clicked on it. But after I opened the file again I must recognize that no changes were saved. I'm disappointed because no warning was showed and my work is lost.

Please implement a warning hint into your app if users are opening .mm files or disable any edit functions if no changes can't be saved.

Perhaps you could add a save-as-feature for .mm files to the .km format in future so my collegues and I could collaborative update existing .mm mindmaps. This would be fantastic.

Thanks in adavance

**Comment by NicolasToussaint** [comment]
I agree that an export / import to `mm` format  feature would be awesome, and bring a lot of freedom in editing the maps in different contexts (Nextcloud vs Desktop for instance)
